compiler/parser/RdrHsSyn.hs

```diff
@@ -2969,7 +2960,7 @@ mkInterfaceMethodDecl ifaceTy classTy methodName methodType mbDocString =
         L _ rdrName = methodName
 
 mkHasInterfaceViewDecl :: Located RdrName -> Maybe (LHsType GhcPs) -> [LHsDecl GhcPs]
-mkHasInterfaceViewDecl iface Nothing = []
+mkHasInterfaceViewDecl _ Nothing = []
```

Comment:
boys scout change to avoid warning

Reply:
# :fleur_de_lis:
github should have a fleur de lis react for stuff like this :)